syntax = "proto3";

import "ClassInfo.proto";
import "Sentence.proto";

package class_system;

message Response {
  optional ClassInfo    class_info = 1;
  repeated Sentence     sentences  = 2;
  string                error_msg  = 4;
}
